test/test_helper.exs

```diff
@@ -42,6 +42,15 @@ Mimic.copy(ExTwilio.Message)
 Mimic.stub(Goth)
 Mimic.stub(Finch)
 
-ExUnit.configure(exclude: [integration: true, failing: true, benchmark: true])
+has_telegraf? = not is_nil(System.find_executable("telegraf"))
```

Comment:
Telegraf is currently not installed on CI, so we will automatically skip these tests. There are some GitHub Actions to install it, but I didn't know what are the policies around adding 3rd party actions, so I have left it for the future.